tools/scripts/list_stl_dependencies.sh

```diff
@@ -22,30 +22,74 @@
 #   ./list_stl_dependencies.sh
 #=====================================================
 
+SCOPE=${1:-list}
 COMPONENTS=(iceoryx_hoofs iceoryx_posh)
 SOURCE_DIR=(source include)
 WORKSPACE=$(git rev-parse --show-toplevel)
+QNX_PLATFORM_DIR=$WORKSPACE/iceoryx_hoofs/platform/qnx/
+USELIST=$WORKSPACE/tools/scripts/used-headers.txt
+TEMP_FILE=$(mktemp)
+GET_HEADER_NAME="\<\K[^<>]+(?=>)" # Matches the content between angle brackets
 
 for COMPONENT in ${COMPONENTS[@]}; do
     for DIR in ${SOURCE_DIR[@]}; do
         GREP_PATH="${GREP_PATH} ${WORKSPACE}/${COMPONENT}/$DIR"
     done
 done
 
+echo "# QNX platform/ libc headers" >> $TEMP_FILE
+
+echo "# usage of headers for QNX"
+gcc -w -fpreprocessed -dD -E $(find $QNX_PLATFORM_DIR -type f -iname *.cpp -o -iname *.hpp) \
+ | grep -e "#include <" \
+ | grep -oP $GET_HEADER_NAME \
+ | sort \
+ | uniq \
+ | tee -a $TEMP_FILE \
+ | cat
+
+
+echo "# iceoryx_posh / iceoryx_hoofs headers" >> $TEMP_FILE
+# GCC can't preprocess .inl so we grep them in plain text
 echo
-echo usage of std components by file
-grep -RIne "std::" $GREP_PATH | sed -n  "s/\([^:]*\:[0-9]*\)\:.*\(std::[a-zA-Z_]*\).*/\ \ \1  \2/p" | sort | uniq
+echo "# usage of <..> header includes"
```

Comment:
Could you please unify the output string and use either `"# iceoryx_posh / iceoryx_hoofs headers"` or `"# usage of <..> header includes"`?

I just ran the script and was confused by this. Same with the QNX header.